ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__arm64-nightly.yaml

```diff
@@ -75,6 +75,12 @@ releases:
       product: ocp
       stream: nightly
       version: "4.17"
+  source:
```

Comment:
openshift-e2e-test-qe does not reference openshift-extended-upgrade-pre, so we do not need the setting here.

Reply:
it has openshift-upgrade-qe-test-heterogeneous which has pre step, so keep it.